lib/matplotlib/tests/test_image.py

```diff
@@ -402,6 +415,7 @@ def test_bbox_image_inverted():
 
     bbox_im = BboxImage(Bbox([[100, 100], [0, 0]]))
     bbox_im.set_data(image)
+    bbox_im.set_clip_on(False)
     axes.add_artist(bbox_im)
```

Comment:
This one is because the original test images were never correct, then got
regenerated wrong.

This is testing that a seg fault is fixed and bboximage should get a better
image test.

On Wed, Jan 6, 2016, 15:52 Benjamin Root <notifications@github.com> wrote:

> In lib/matplotlib/tests/test_image.py
> <https://github.com/matplotlib/matplotlib/pull/5718#discussion_r49007890>:
>
> > @@ -402,6 +415,7 @@ def test_bbox_image_inverted():
> >
> >      bbox_im = BboxImage(Bbox([[100, 100], [0, 0]]))
> >      bbox_im.set_data(image)
> > +    bbox_im.set_clip_on(False)
> >      axes.add_artist(bbox_im)
>
> Why drop the image_comparison here? This test is now just a smoketest.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5718/files#r49007890>.
>